src/main/groovy/com/morpheusdata/commvault/backup/CommvaultBackupTypeProvider.groovy

```diff
@@ -152,7 +158,7 @@ class CommvaultBackupTypeProvider extends AbstractBackupTypeProvider {
 	@Override
 	CommvaultBackupExecutionProvider getExecutionProvider() {
```

Comment:
After converting this class to an abstract class the methods `getExecutionProvider()` and `getRestoreProvider()` should be left out of this class to force the implementing class to implement the methods.

Reply:
@ddeyoung changes are made as suggested